go.mod

```diff
@@ -1,63 +1,35 @@
-module github.com/twilio-labs/sample-twilio-go
+module code.hq.twilio.com/twilio/review-rewards-example-app
```

Comment:
We should change this to GH as well.

Reply:
Changed the module name to the new repo location; `github.com/twilio-labs/sample-twilio-go`